### DISCUSSION TITLE
Relay support for filter??

### DISCUSSION THREAD

**Discussion by 0xHexE** [ChilliCream/hotchocolate#5664] [Q&A]
Hi,

What is a better option in filter id be base64 encoded, or should it be UUID?

![image](https://user-images.githubusercontent.com/29942725/211166935-4c3a828e-441a-4386-945e-ff43eae69863.png)

**Comment by PascalSenn** [comment]
In Version 13 we automatically infer `ID` on the type and generate a `IdOperationFilterInputType` 
```csharp
public class Customer
{
    [ID]
    public int Id { get; set; }
}
```

```graphql
{
  customers(where: {id: {eq:"Q3VzdG9tZXIKaTE="}}) {
    id
  }
}

```
